src/tox_uv/_venv.py

```diff
@@ -83,6 +85,16 @@ def _get_python(self, base_python: list[str]) -> PythonInfo | None:  # noqa: PLR
                     platform=sys.platform,
                     extra={},
                 )
+            if os.path.isabs(base):  # noqa: PTH117
+                info = cached_py_info.from_exe(cached_py_info.PythonInfo, None, base)
```

Comment:
We should pass in the cache path to be the virtualenv one.

Reply:
Can you provide more detail please? I don't see an obvious arg to `from_exe()`... is it `app_data`? How do I get the correct value? I see [this guy here](https://github.com/pypa/virtualenv/blob/main/src/virtualenv/app_data/__init__.py#L16-L20), but it's a private API.

Reply:
Amended commit to use `virtualenv.app_data.make_app_data()`, assuming that this is what you were meaning.